silk-plugins/silk-serialization-json/src/main/scala/org/silkframework/serialization/json/JsonSerializers.scala

```diff
@@ -259,17 +258,17 @@ object JsonSerializers {
       ValueType.valueTypeById(nodeType) match {
         case Left(_) =>
           nodeType match {
-            case "CustomValueType" =>
+            case ValueType.OUTDATED_AUTO_DETECT => UntypedValueType
```

Comment:
As already mentioned above, the migration of the "auto detect" type should be discussed.